tests/unit/rules/javascript/test.unsupported_browser_api.js

```diff
@@ -121,4 +121,96 @@ describe('unsupported browser APIs', () => {
     const { linterMessages } = await runJsScanner(jsScanner);
     expect(linterMessages.length).toEqual(0);
   });
+
+  describe('Experiement APIs', () => {
+    it.each([
+      {
+        title: 'allows known privileged APIs when the extension is privileged',
+        privileged: true,
+        expectedMessage: null,
+      },
+      {
+        title: 'emits an error when the extension is not privileged',
```

Comment:
Not a big deal because the categorization between errors and warnings does not happen at the JavascriptScanner level, but given that technically [UNSUPPORTED_API is categorized as a warning](https://mozilla.github.io/addons-linter/#web-extensions-%2F-manifest.json) we may change `emits an error` into  `emits a warning` if we prefer to reflect that in the tests descriptions.